src/egon/data/datasets/etrago_setup.py

```diff
@@ -97,6 +97,7 @@ class EgonPfHvGenerator(Base):
     ramp_limit_down = Column(Float(53), server_default="NaN")
     ramp_limit_start_up = Column(Float(53), server_default="1.")
     ramp_limit_shut_down = Column(Float(53), server_default="1.")
+    e_nom_max = Column(Float(53), server_default="inf")
```

Comment:
Maybe add a comment here, why we add e_nom_max as columns, since it is not part of the generators default columns.